cttportfolio.css

```
@@ -3,168 +3,481 @@
-	position:sticky;
+	position:fixed;
 	top:0;
 	width:100vw;
-	height:50px;
+	z-index:1;
+	margin:10px 0;
+}
+nav a, nav p{
+	text-decoration:none;
+	color:rgba(248,248,255, 1);
+	text-transform:uppercase;
+}
+.navlogo > img{
+	padding-right:15px;
+	height:100%;
+	filter: invert(0.95); 
+	position:relative;
+	bottom:4px;
 }
-nav > a > img{
-	height:42px; 
+
+.navlogo > p{
+	font-size:2.5rem;
+	font-weight:700;
+}
+.navlogo{
+	display:flex;
+	align-items:center;
+	height:80px;
+	
+}
+.navlinks > a{
+	font-size:2rem;
+	margin-left:30px;
+	padding:15px 10px;
+	padding-top:18px;
+	border:1px solid rgba(248,248,255,0);
+	border-radius: 5px;
+	font-weight:600;
+}
+.navlinks a:hover{
+	border:1px solid rgba(248,248,255,1);
+
 }
 /*======================================== HEADER SECTION ========================================*/
 header{
-	background-color:#808080;
 	width:100vw;
-	height:calc(100vh - 50px);
+	height:100vh;
+	position:relative;
+	z-index:-1;
+}
+#frontpagevideo{
+	object-fit:cover;
+	width:inherit;
+	max-height:inherit;
+	position:fixed;
+}
+.headertext{
+	position:absolute;
+	top:0;
+	left:0;
+	width:inherit;
+	height:inherit;
 	display:flex;
-	align-items:center;
+	flex-direction:column;
 	justify-content:center;
+	padding-left:20%;
 }
-/*======================================== ABOUT ME SECTION ========================================*/
-.aboutmediv{
-	background-color:#1f7a1f;
+h1{
+	font-size:2.5rem;
+	color:white;
+}
+h2{
+	font-size:1.8rem;
+	text-transform:uppercase;
+	color: rgb(248,248,255);
 }
-.aboutmemain{
+/*======================================== ABOUT ME SECTION ========================================*/
+.aboutme{
+	height:1200px;
+	width:100%;
 	display:flex;
 	flex-direction:column;
-	align-items:center;
+	justify-content:space-between;
 }
-.aboutmemainchild{
-	background-color:#33cc33;
-	display:flex;
+.aboutme > * {
+	margin-bottom:50px;
+	border-radius:10px;
+}
+.aboutme > button{
+	margin:0 auto;
+}
+.amcontainer{
 	width:100%;
+	height:500px;
+	display:flex;
+}
+.am_main > *{
+	padding:50px;
+}
+.am_img{
+	width:30%;
+}
+.am_img > img{
+	max-width:100%;
+	max-height:100%;
+	border-radius:50%;
+}
+/*======================================== SKILLS SECTION ========================================*/
+.am_sub{
+	margin-top:50px;
+}
+.am_sub_img{
+	position:relative;
 }
-.aboutmeimg{
-	width:25%;
+#am_sub_ar{
+	opacity:0;
+}
+.am_sub_content{
+	position:absolute;
+	top:0;
+	left:0;
+	right:0;
+	bottom:0;
+	padding:50px;
+}
+.sub_img{
+	max-width:100%;
+	height:100%;
+	object-fit:cover;
+	border-radius:50%;
+}
+.am_sub .am_text{
+	padding:50px;
+}
+.am_text{
+	background-color:rgba(255,182,193,0.6);
+	width:70%;
 	display:flex;
+	flex-direction:column;
 	justify-content:center;
+	margin:auto 0;
+	padding-top:40px;
 }
-
-.aboutmeimg>img{ /*634x951 this is teh stock image */
-	max-width:634px;
-	max-height:951px;
-	width:inherit;
-	height:auto;
-}
-.aboutmetext{
-	padding-right:15%;
+.am_text > p{
+	font-size:2.0rem;
```

You don't need the `.0`.
```suggestion
	font-size:2rem;
```